packages/react-native/lib/wallet.ts

```diff
@@ -0,0 +1,29 @@
+import {createWallet, IWallet} from '@docknetwork/wallet-sdk-core/src/wallet';
+import {dockDocumentNetworkResolver} from '@docknetwork/wallet-sdk-core/src/network-resolver';
+
+let wallet: IWallet;
+
+export function getWallet() {
+  if (!wallet) {
+    throw new Error('Wallet not initialized');
+  }
+
+  return wallet;
+}
+
+export async function getOrCreateWallet() {
+  if (!wallet) {
+    wallet = await initializeWallet();
+  }
+
+  return wallet;
+}
+export async function initializeWallet() {
+  wallet = await createWallet({
+    databasePath: 'dock-wallet',
+    dbType: 'react-native',
```

Comment:
I wouldn't expect a `dbType` to be `react-native`. I was expecting a database related value like postgres or mysql or sqlite or something

Reply:
I agree, but this is the constant value defined on `typeorm` side. https://github.com/typeorm/react-native-example/blob/master/src/App.tsx#L42

`react-native` is used for mobile a sqlite, and is mentioned on their docs. For nodejs `sqlite` should be used. We could map it out on our side, but could create breaking changes and not be consistent with `typeorm` docs

Reply:
ah, that's an odd choice on their part

no need to complicate things on our side